lightning/src/ln/channel.rs

```diff
@@ -9506,8 +9506,7 @@ impl<SP: Deref> Writeable for Channel<SP> where SP::Target: SignerProvider {
 			(47, next_holder_commitment_point, option),
 			(49, self.context.local_initiated_shutdown, option), // Added in 0.0.122
 			(51, is_manual_broadcast, option), // Added in 0.0.124
-			(53, funding_tx_broadcast_safe_event_emitted, option), // Added in 0.0.124
-			(55, self.context.next_funding_txid, option) // Added in 0.1.0
+			(53, funding_tx_broadcast_safe_event_emitted, option) // Added in 0.0.124
```

Comment:
nit: It's better to keep the ending comma on the last line as well.

Reply:
I'd just like to revert it back to how it existed before the change where there was no trailing comma after type 53.

Reply:
Hmm it seems this was one of the few places where trailing commas weren't consistently added, so in that case I'll just add it back so it's less noisy with new additions even though it's not a pure revert.